src/server/api/mutations/joinOrganization.js

```diff
@@ -26,6 +26,20 @@ export const joinOrganization = async (
       organization = await cacheableData.organization.load(
         campaign.organization_id
       );
+      const maxTextersPerCampaign = getConfig(
+        "MAX_TEXTERS_PER_CAMPAIGN",
+        organization
+      );
+      if (maxTextersPerCampaign) {
+        const campaignTexterCount = await r.getCount(
+          r.knex("assignment").where("campaign_id", campaignId)
+        );
+        if (campaignTexterCount >= maxTextersPerCampaign) {
+          throw new GraphQLError(
```

Comment:
If you throw a regular `Error` here it should simplify the error handling in JoinTeam.  I think  GraphQLError is an internal type that isn't meant to be used in the application. apollo-server 2 will give us better tools for this.

Reply:
This is not a big deal, going to merge this now